Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -312,7 +312,17 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
         }
     }()
 
-    public lazy var tunnelFailureMonitor = NetworkProtectionTunnelFailureMonitor(tunnelProvider: self)
+    lazy var deviceManager: NetworkProtectionDeviceManagement = NetworkProtectionDeviceManager(
+        settings: self.settings,
+        tokenStore: self.tokenStore,
+        keyStore: self.keyStore,
+        errorEvents: self.debugEvents,
+        isSubscriptionEnabled: self.isSubscriptionEnabled
+    )
+
+    private lazy var tunnelFailureMonitor = NetworkProtectionTunnelFailureMonitor(handshakeReporter: adapter)
```

Comment:
I didn't understand this comment.  What was the alternative to using `NetworkProtectionTunnelFailureMonitor` and how would that alternative have affected the thresholds?

Reply:
Another option would be to make the monitor configurable so that the thresholds are not hard-coded. We (Karl / Aitor / I) decided that we should ship this and see how it’s performing relative to Android then perhaps tweak the threshold if need be.

Reply:
Thanks for the context.

The option of injecting is always present and doesn't need to be explained (IMO), so I think removing the comment, as you did, makes sense.